x-pack/dockerlogbeat/handlers.go

```diff
@@ -70,7 +70,7 @@ func stopLoggingHandler(pm *pipelinemanager.PipelineManager) func(w http.Respons
 		go func() {
 			err = pm.CloseClientWithFile(stopReq.File)
 			if err != nil {
-				pm.Logger.Infof(" Got stop request error %#v\n", err)
+				pm.Logger.Errorf(" Got stop request error %#v\n", err)
```

Comment:
Is this space intended?
```suggestion
				pm.Logger.Errorf("Got stop request error %#v\n", err)
```